test/metabase/query_processor_test/unix_timestamp_test.clj

```diff
@@ -9,16 +9,21 @@
              [datasets :as datasets :refer [*driver* *engine*]]
              [interface :as i]]))
 
-;; There were 9 "sad toucan incidents" on 2015-06-02
+;; There were 10 "sad toucan incidents" on 2015-06-02 in UTC
```

Comment:
Manually validated this at the repl from the data files directly (no database). As UTC there are 10 events in UTC on 6/2. In pacific time there are 9. MySQL is the only test database in pacific time, and that is being overridden by our usage of the report timezone here. All results should be 10 now.